container/sagemaker/tensorflow-serving.js

```diff
@@ -168,38 +168,21 @@ function json_lines_request(r, data) {
 function csv_request(r) {
     var data = r.requestBody
 
-    // look for initial quote or numeric-only data in 1st field
-    var needs_quotes = data.search(/^\s*("|[\d.Ee+\-]+\s*,)/) != 0
-
     var lines = data.trim().split(/\r?\n/)
 
-    // start instances json - omit outer list for single example
-    var json = '{"instances":'
-    if (lines.length != 1) {
-        json += '['
-    }
+    var builder = []
+    builder.push('{"instances":[')
```

Comment:
assembling an array and then joining is cleaner code, but I think it performs worse and uses more memory. can you try the string append way and do a comparison test?

Reply:
Yeah, I put the ab results at the end of the PR description -- it's basically the same as string append. I'll rerun the other benchmarks after I make the change below, which might change this. (This was surprising since resources online said += was faster, but I guess that's only for various browsers and not for nginx?)

Reply:
ya i found the same sources when i did the initial implementation. this is why i think we ought to test. :)